learn/pkg/learn/question.go

```diff
@@ -548,6 +657,10 @@ func (m *QuestionModel) isParserErrorQuestion() bool {
 	return m.Frontmatter.Verification == "parse-error" || m.Frontmatter.Verification == "no-parse-error"
 }
 
+func (m *QuestionModel) isTextQuestion() bool {
```

Comment:
this function is open-coded in one other place (although in that place I did suggest replacing it with a switch). Either way use the function everywhere or nowhere I think

Reply:
ok, removed